fhir-server/src/main/java/com/ibm/fhir/server/resources/Capabilities.java

```diff
@@ -91,6 +92,8 @@
     private static final String FHIR_SERVER_NAME = "IBM FHIR Server";
     private static final String FHIR_COPYRIGHT = "(C) Copyright IBM Corporation 2016, 2020";
     private static final String EXTENSION_URL = "http://ibm.com/fhir/extension";
+    private static final String BASE_CAPABILITY_URL = "http://hl7.org/fhir/CapabilityStatement/base";
+    private static final String BASE_2_CAPABILITY_URL = "http://hl7.org/fhir/CapabilityStatement/base2";
```

Comment:
what are these for?

Reply:
The FHIR registry includes these two capability statements:
/fhir-registry/src/main/resources/hl7/fhir/core/package/CapabilityStatement-base.json
/fhir-registry/src/main/resources/hl7/fhir/core/package/CapabilityStatement-base2.json
Since they seemed related to the base FHIR spec and have experimental = true, I wasn't sure they should be included in "instantiates".

Reply:
interesting.  i agree we should exclude them.  i wonder whether they should truly be in the registry at all.
@JohnTimm thoughts?

Reply:
if we don't here from john by the morning, I think we should just add a comment (either here or where I just suggested it below) and merge this one in time for 4.4.1